rpcs3/rpcs3qt/pad_settings_dialog.cpp

```diff
@@ -928,6 +937,39 @@ void pad_settings_dialog::ChangeInputType()
 	m_handler = GetHandler(g_cfg_input.player[player]->handler);
 	const auto device_list = m_handler->ListDevices();
 
+	// Change the description
+	QString description;
+	switch (m_handler->m_type)
+	{
+	case pad_handler::null:
+		description = m_json_handlers["null"].toString(); break;
+	case pad_handler::keyboard:
+		description = m_json_handlers["keyboard"].toString(); break;
+#ifdef _WIN32
+	case pad_handler::xinput:
+		description = m_json_handlers["xinput"].toString(); break;
+	case pad_handler::mm:
+		description = m_json_handlers["mmjoy"].toString(); break;
+	case pad_handler::ds3:
+		description = m_json_handlers["ds3_windows"].toString(); break;
+	case pad_handler::ds4:
+		description = m_json_handlers["ds4_windows"].toString(); break;
+#else
+	case pad_handler::ds3:
+		description = m_json_handlers["ds3_linux"].toString(); break;
+	case pad_handler::ds4:
+		description = m_json_handlers["ds4_linux"].toString(); break;
+#endif
+#ifdef HAVE_LIBEVDEV
+	case pad_handler::evdev:
+		description = (m_json_handlers["evdev"].toString()); break;
+#endif
+	default:
+		LOG_ERROR(GENERAL, "Description missing for handler %s", handler);
```

Comment:
There's no need to throw an error for something this unimportant.
You'll see an empty description anyway.

Reply:
I thought it'd be better to have an error, but I guess you're right.